problems/objects.js

```diff
@@ -168,16 +222,39 @@ const cubeObj = () => {};
  * @returns {Object} Counts of e and a. {a: 2, e: 1}
  */
 
-const countAandE = () => {};
+const countAandE = (str) => {
+  let obj = {a: 0, e: 0}
+  let array = str.split("")
```

Comment:
Remember you can loop through a string, you don't need to turn it into a split array